styleguide/derek/_partials/_rug.ejs

```diff
@@ -1,12 +1,48 @@
 <!--[rug]-->
-<div class="container rug">
+<div class="container-fluid bg-blue-gray half-padding">
+<div class="container rug hidden-xs hidden-sm">
```

Comment:
Should the `hidden-xs hidden-sm` be moved to the `container-fluid` wrapper? I am still seeing a dark gray stripe at the bottom of the page in narrow viewport widths.